setup_dashboard_and_es_ism.sh

```diff
@@ -267,3 +267,19 @@ curl -X PUT "$ES_ENDPOINT/opni-drain-model-status-000001" --insecure -u "$ES_USE
 }
 '
 
+curl -X PUT "$ES_ENDPOINT/opni-normal-intervals" --insecure -u "$ES_USER:$ES_PASSWORD" -H 'Content-Type: application/json' -d'
```

Comment:
Seems like this should be ES_USERNAME based on the environment variables used by the services.

Reply:
why does that matter?

Reply:
this script is run with `ES_USER` (and other) env set so it doesn't need to be modified